theano/compile/tests/test_function_module.py

```diff
@@ -266,7 +279,7 @@ def test_copy_share_memory(self):
             for key in storage_map_cpy.keys():
                 storage = storage_map_cpy[key]
                 if key not in i_o_variables or isinstance(key, theano.tensor.Constant):
-                    self.assertTrue(any([ storage is s for s in ori_storages]))
+                    self.assertTrue(any([storage is s for s in ori_storages]))
```

Comment:
If I'm not wrong this is just a very very weird way to do:
```python
self.assertTrue(storage in ori_storages)
```

More generally, with @lamblin or @abergeron benediction, I would rewrite [this whole for loop](https://github.com/olimastro/Theano/blob/0440ea6e2ec912b0c54adddaefcc6d67036e18bc/theano/compile/tests/test_function_module.py#L279-L282) as:
```python
l = [val for key,val in storage_map_cpy.items() 
     if key not in i_o_variables or isinstance(key, theano.tensor.Constant)]
assert all([val in ori_storages for val in l])
```